invokeai/app/services/board_records/board_records_common.py

```diff
@@ -22,6 +22,8 @@ class BoardRecord(BaseModelExcludeNull):
     """The updated timestamp of the image."""
     cover_image_name: Optional[str] = Field(default=None, description="The name of the cover image of the board.")
     """The name of the cover image of the board."""
+    archived: Optional[bool] = Field(default=None, description="Whether or not the board is archived.")
```

Comment:
Shouldn't this always be populated? If so, it probably should be required. Would allow us to skip being concerned with it being unvalued in the frontend